dashboard/app/models/ability.rb

```diff
@@ -92,7 +92,7 @@ def initialize(user)
         can [:read, :find], :regional_partner_workshops
         can [:new, :create, :read], Pd::WorkshopMaterialOrder, user_id: user.id
         can [:new, :create, :read], Pd::Application::Facilitator1819Application, user_id: user.id
-        can [:new, :create, :read], Pd::Application::Teacher1819Application, user_id: user.id
+        can [:new, :create, :read], Pd::Application::Teacher1920Application, user_id: user.id
```

Comment:
I'm surprised a little by this - have we had this bug for some time that was blocking creation of teacher application?

Reply:
I was also surprised. It worked because the controller was checking auth on the old version too. See https://github.com/code-dot-org/code-dot-org/pull/25282/files#diff-1407e3d08865bd76f5e75007ec39adafL6